src/utils/__tests__/createRandomArray.test.ts

```diff
@@ -31,4 +33,25 @@ describe('createRandomXArray', () => {
       expect(y).toBeLessThan(10500);
     }
   });
+
+  it('Testing in conjunction with spectra-fitting', () => {
+    let array = createRandomArray({
+      mean: 10,
+      range: 2,
+      length: 100000,
+      distribution: 'uniform',
```

Comment:
Is it normal you try to fit with a gaussian an uniform distribution ? It is not expected to be a flat histogram ?

Did you check the histogram ? It is strange you add testcases in which there are only `0` decimals that are correct. I would expect a better precision.

Reply:
Yes I figured the code is wrong, the precision is now 3